components/table/README.md

```diff
@@ -134,9 +134,9 @@ For long tables, the header row can be made to "stick" in place as the user scro
 
 When tabular data can be sorted, the `<d2l-table-col-sort-button>` can be used to provide an interactive sort button as well as arrows to indicate the ascending/descending sort direction. This is meant to be used within a `d2l-table-wrapper`.
 
-Note that the example below hides much of the implementation. See the code in [Multi-Faceted Sort Button](#multi-faceted-sort-button) for a more detailed implementation example.
+Note that the example below hides the implementation, and also that the code is handling the `desc` attribute so updating it in the Properties table will not have an impact on the live demo. See the code in [Multi-Faceted Sort Button](#multi-faceted-sort-button) for a more detailed implementation example.
```

Comment:
I don't know what it is exactly, but the first sentence just feels a bit clunky to me. It might be good to have it be split into two sentences instead.